.github/workflows/test.yml

```diff
@@ -788,6 +788,89 @@ jobs:
         with:
           commit_message: Benchmark result for ${{ steps.date.outputs.pretty }} (${{ github.sha }})
 
+  next_js_integration:
```

Comment:
Should this get added to the `final` job?

Reply:
I'd not for now - it is unclear how we'll treat this test results for the CI yet between blocking vs. not, and what to do if we start to enable actual turbo tests.